docs/index.md

```diff
@@ -7,15 +7,9 @@ https://docs.python-guide.org/writing/structure/). Whisk does the structuring wh
 
 Read more about our [beliefs](#beliefs).
 
-## A quick tour of whisk
+## Getting Started
```

Comment:
@itsderek23 I didn't want to reference "quick tour" multiple times in different places, so I decided to rename this to "getting started" - let me know what you think.

Reply:
Looks good